### PR TITLE
Update to Rook v1.5

### PR DESCRIPTION
**Pull request by pkoutsov**
This PR updates the yaml files in ```plugins/ceph-cache-plugin/deploy/rook``` to the Rook v1.5. Also, for testing purposes, it presents how to set up a ceph cluster with csi-lvm (```plugins/ceph-cache-plugin/deploy/rook/cluster-on-pvc.yaml```)

### CODE REVIEW

File: docs/Ceph-Caching.md [hunk 1]
```diff
@@ -7,20 +7,27 @@ First we need to have a working cluster.
 `minikube start --memory='10G' --cpus=8 --disk-size='15g' --driver=docker`
```
Comment (YiannisGkoufas) [inline docs/Ceph-Caching.md:7]: Maybe have the part 2 be minikube only and hardcode the values that make sense both for the above (disk-size)

Comment (pkoutsov) [reply docs/Ceph-Caching.md]: implemented

File: docs/Ceph-Caching.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
 **The instructions currently work only for the branch https://github.com/IBM/dataset-lifecycle-framework/tree/fixed-caching**
```
Comment (YiannisGkoufas) [inline docs/Ceph-Caching.md:1]: remove this as well

Comment (pkoutsov) [reply docs/Ceph-Caching.md]: removed

**Comment by YiannisGkoufas** [comment]
looks good @pkoutsov !

**Pull request merged**